doc/telescope.txt

```diff
@@ -502,11 +502,6 @@ builtin.tags({opts})                                          *builtin.tags()*
         {show_line}     (boolean)  if true, shows the content of the line the
                                    tag is found on in the picker (default is
                                    true)
-        {shorten_path}  (boolean)  if true, makes file paths shown in picker
```

Comment:
please don't modify `doc/telescope.txt` by hand. Its autogenerated. You need to adjust `lua/telescope/builtin/init.lua`

And than you need to make sure that " Allow all actions " and " Read and write permissions " are set in your fork settings, actions.

Reply:
Not sure what the purpose of those repo settings, but they are set already. I'll fix the docs though.

Reply:
I am just mentioning it because the CI should run docgen and revert the changes. But we have a lot of problems with that lately. Here it worked https://github.com/nvim-telescope/telescope.nvim/pull/888 and here it didn't https://github.com/nvim-telescope/telescope.nvim/pull/887

:|